### PR TITLE
chore: move the astyle repo to the lvgl organization

### PR DESCRIPTION
**Pull request by kisvegabor**
### Description of the feature or fix

A clear and concise description of what the bug or new feature is.

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) version [v3.4.12](https://github.com/szepeviktor/astyle/releases/tag/v3.4.12) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

**Comment by lhdjply** [comment]
The PR template needs to be modified
https://github.com/lvgl/lvgl/blob/93e01893096b40edc505ba72df571274744ba263/.github/pull_request_template.md#L10

File: scripts/install_astyle.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+rm -rf astyle
+git clone https://github.com/lvgl/astyle.git
```
Comment (cristian-stoica) [inline scripts/install_astyle.sh:10]: Is the lvgl/astyle.git going to be frozen to a particular commit or is it better to checkout a particular version here? (Or maybe it doesn't matter?)

Comment (kisvegabor) [reply scripts/install_astyle.sh:10]: We can update `install_astyle.sh` to install the correct version. Using `master` is fine now.

**Review by lhdjply — APPROVED** [review]

File: .github/workflows/check_style.yml [hunk 2]
```diff
@@ -16,9 +16,8 @@ jobs:
       - name: Checkout astyle
         uses: actions/checkout@v4
         with:
-          repository: szepeviktor/astyle
+          repository: lvgl/astyle
           path: astyle
-          ref: v3.4.12
```
Comment (XuNeo) [inline .github/workflows/check_style.yml:21]: Should we use specific version instead. The developers can easily check version that CI uses from here. Using `master` is not recommended.

**Comment by kisvegabor** [comment]
Now that two of you said that we should use a specific version I added it :smile:

**Review by XuNeo — APPROVED** [review]

**Review by liamHowatt — APPROVED** [review]

**Pull request merged**